### ISSUE TITLE
Use new card icon components in CardNumberInput

### ISSUE THREAD

**Issue by felixjung** [sumup/circuit-ui#209]
#190 introduced new components for card scheme icons. We should update the `CardNumberInput` component to use those icons.

**Comment by robinmetral** [comment]
Hey @felixjung :wave: Is this issue still relevant? I can take care of it!

**Comment by felixjung** [comment]
Hey @robinmetral, that's awesome! Thanks for offering. Unfortunately, we need to do some cleanup before we can start accepting outside contributions, again. As you may have noticed we don't have a LICENSE at the moment and we don't have a contributor license agreement (CLA) in place.

I'm looking to add the license _very_ soon, but the CLA and a way to accept it will take until early June, unfortunately. I know it's bad and we should put that information out there. Sorry about that. 

![GIF of disappointed kid](https://media.giphy.com/media/2m21KnBOB9TUCAwBRz/giphy.gif)

It was really really great for you to offer to help. Keep it up! 👍 The issue itself is still relevant and as soon as we are ready to accept contributions you or someone else can take it.

**Comment by robinmetral** [comment]
Thanks for your quick reply @felixjung!
I understand and I'll happily wait until I can sign your CLA in June :slightly_smiling_face: 

(or I can just join SumUp as a Frontend Engineer and contribute from the inside :stuck_out_tongue: )

![gif](https://user-images.githubusercontent.com/35560568/56208063-882fe080-6050-11e9-878e-c56a446d6066.gif)

**Comment by connor-baer** [comment]
Closing, since this component is deprecated and will be removed in the next major version (see #469).
